World/WeatherManager.gd

```diff
@@ -40,25 +51,38 @@ var wind_direction = 0 :
 		return wind_direction
 	set(new_wind_direction):
 		wind_direction = new_wind_direction
-		emit_signal("wind_direction_changed", wind_direction)
+		wind_direction_changed.emit(wind_direction)
 
 var rain_enabled := false :
 	get:
 		return rain_enabled
 	set(enabled):
 		rain_enabled = enabled 
-		emit_signal("rain_enabled_changed", enabled)
+		rain_enabled_changed.emit(enabled)
 
-var rain_density := 100.0 :
+var rain_density := 1 :
```

Comment:
Also add documentation for sensible range of values here; also, maybe a rain_density of 0 is enough to imply that rain is disabled, and we can therefore remove `rain_enabled` above? (would avoid ambiguity/confusion when setting the rain_density but forgetting to set rain_enabled)

Reply:
I agree with the range, however I am not sure if I would want to remove rain_enabled. The `Rain` node is - to a large extent - just a group of `GPUParticles`. `GPUParticles` in Godot are closely resembled by and `enabled` and a `density` via their properties `emitting` and `amount`. That is why I personally designed it this way. Do you think the avoidance of ambiguity has higher priority?